cfme/tests/infrastructure/test_delete_infra_object.py

```diff
@@ -127,3 +128,15 @@ def test_delete_cluster_from_table(setup_provider, provider):
     cluster1 = cluster.Cluster(name=cluster_name, provider=provider)
     collection = ClusterCollection()
     collection.delete(cluster1)
+
+
+def test_delete_resource_pool_from_table(setup_provider, provider):
+    """ Tests delete resource pool from table
+
+    Metadata:
+        test_flag: delete_object
+    """
+    resourcepool_name = provider.data['remove_test']['resource_pool']
+    resourcepool1 = resource_pool.ResourcePool(name=resourcepool_name)
```

Comment:
**QUESTION**: why ```resourcepool1```? Also, you should use the collections.instantiate to get the object, and not refer to the object directly.